test/tests/ver-1dc/tests

```diff
@@ -0,0 +1,37 @@
+[Tests]
+  design = 'TrappingNodalKernel.md ReleasingNodalKernel.md'
+  issues = '#12'
+  verification = 'ver-1dc.md'
+  [ver-1dc_limited]
```

Comment:
I don't have time now, but I'll need to confirm that this case runs in under 2 seconds.

Reply:
Usually, how will you calclate the running time of the test cases? I just run 
`time ~/projects/TMAP8/tmap8-opt -i ver-1dc.i nx_num=20 Executioner/num_steps=300 Outputs/file_base=ver-1dc_limited_test_exodus`
on my terminal and I got the running time here:
```
real	0m0.825s
user	0m0.599s
sys	0m0.206s
```

Is it the common method to calculate the running time of test case?

Reply:
You can also use `./run_tests -j2 --re=ver-1dc_limited` and see the time listed. If the maximum test time is under 2 seconds, you are good!

In this case, you are good!